scripts/devops_tasks/test_run_samples.py

```diff
@@ -99,9 +99,7 @@
         "mgmt_subscription_async.py",
         "mgmt_topic_async.py",
         "proxy_async.py",
-        "receive_deferred_message_queue_async.py",
-        "authenticate_using_azure_sas_credential_async.py",
-        "authenticate_using_azure_sas_credential.py"
+        "receive_deferred_message_queue_async.py"
```

Comment:
does async sample `authenticate_using_azure_sas_credential_async` need update?

Reply:
I don't believe so? I added the env vars to the template, so both the sync and async samples should run and pass in the pipeline.